x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-rds-overview.json

```diff
@@ -0,0 +1,786 @@
+      "updated_at": "2019-06-14T17:02:27.742Z",
+      "version": "WzI2NzQsNF0="
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "AWS RDS Insert Throughput",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [],
+          "params": {
+            "axis_formatter": "number",
+            "axis_position": "left",
+            "axis_scale": "normal",
+            "background_color_rules": [
+              {
+                "id": "28cacdf0-921c-11e9-badf-4b42bd1ef543"
+              }
+            ],
+            "bar_color_rules": [
+              {
+                "id": "f8196690-921a-11e9-badf-4b42bd1ef543"
+              }
+            ],
+            "default_index_pattern": "metricbeat-*",
+            "default_timefield": "@timestamp",
+            "id": "61ca57f0-469d-11e7-af02-69e470af7417",
+            "index_pattern": "",
+            "interval": "1m",
+            "series": [
+              {
+                "axis_position": "right",
+                "chart_type": "line",
+                "color": "#68BC00",
+                "fill": "0",
+                "formatter": "number",
+                "id": "61ca57f1-469d-11e7-af02-69e470af7417",
+                "label": "Insert Throughput Count/Second",
+                "line_width": 1,
+                "metrics": [
+                  {
+                    "field": "aws.rds.throughput.insert",
+                    "id": "61ca57f2-469d-11e7-af02-69e470af7417",
+                    "type": "avg"
+                  }
+                ],
+                "point_size": 1,
+                "separate_axis": 0,
+                "split_mode": "terms",
+                "stacked": "none",
+                "terms_field": "aws.rds.db_instance.identifier",
+                "terms_order_by": "61ca57f2-469d-11e7-af02-69e470af7417"
+              }
+            ],
+            "show_grid": 1,
+            "show_legend": 1,
+            "time_field": "",
+            "type": "timeseries"
+          },
+          "title": "AWS RDS Insert Throughput",
+          "type": "metrics"
+        }
+      },
+      "id": "c1afd130-921e-11e9-aa19-159bf182e06f",
+      "migrationVersion": {
+        "visualization": "7.2.0"
+      },
+      "references": [],
+      "type": "visualization",
+      "updated_at": "2019-06-18T23:14:00.771Z",
+      "version": "WzI5NTEsNl0="
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "AWS RDS Select Throughput",
```

Comment:
Is queries per second the unit of these throughputs? We could probably consider `Queries Per Second` visualization also throughput, but they have inconsistent titles. If all them are throughput in queries per second, could we use consistent naming like `Throughput in queries per second`, `Select throughput in queries per second`...

Reply:
Yes, the unit is `Count/Second` which is the same as `Queries per Second` :-) But I just realized this `Queries` metric is only for Aurora-MySQL. I will remove this from the overview dashboard since it is not a general metric for all types of databases.